triton-vm/src/stark.rs

```diff
@@ -558,7 +574,7 @@ impl Stark {
     }
 
     fn shift_codeword(
-        fri_x_values: &Vec<XFieldElement>,
+        fri_x_values: &Vec<BFieldElement>,
```

Comment:
Consider renaming variable `fri_x_values` to `domain_values`, or even passing the `ArithmeticDomain` object rather than the list of elements.

Also, consider changing the type to slice for both `fri_x_values` and `codeword`.